jxbrowser/src/org/opensim/javabrowser/jxBrowserTopComponent.java

```diff
@@ -55,11 +55,23 @@
 
     public jxBrowserTopComponent() {
         initComponents();
-        
+        String useGPU = "On";        
+        String savedGPU = Preferences.userNodeForPackage(TheApp.class).get("GPU Acceleration", useGPU);
+        Preferences.userNodeForPackage(TheApp.class).put("GPU Acceleration", savedGPU);
+        String useLightWeight = "Off";        
+        String savedLightWeight = Preferences.userNodeForPackage(TheApp.class).get("LightWeight Browser", useLightWeight);
+        Preferences.userNodeForPackage(TheApp.class).put("LightWeight Browser", savedLightWeight);
```

Comment:
> these are mainly for development purposes, if either proves useful we should change the name to something meaningful to users indeed

Thanks for explaining. It's fine with me if we use these names but we could also consider naming these "GPU Acceleration (advanced)" or something.